### DISCUSSION TITLE
Affine matrix error

### DISCUSSION THREAD

**Discussion by Leeyeonju1** [Project-MONAI/MONAI#5175] [General]
![image](https://user-images.githubusercontent.com/37902440/191070979-5b9d1790-3834-45e9-a801-57226c3b2ccc.png)

I've got an error like above image. This is 3D data segmentation task and  I'm trying to load several 2d images and  concat them together as 1 multi-channels image, so it is expected to be 3D image. Could someone recommend the solutions?

Below is my code.
![image](https://user-images.githubusercontent.com/37902440/191071875-2b14a8e1-f747-4096-a33d-a8e1528a1fee.png)

Thanks in advance!

**Comment by KumoLiu** [comment]
Hi @Leeyeonju1, from your description, it seems you need to load several 2d images but you just load them from one key 'image'? Maybe you need to load and concat them like this:
```
train_transform  = Compose(
    [
         LoadImaged(keys=['img1', 'img2', 'img3', 'label'], ensure_channel_first=True),
         ConcatItemsd(keys=['img1', 'img2', 'img3'], name='image'),
         ......
]
)
```
Hope it can help you! If you still face problems, feel free to ask me!
Thanks!
